### ISSUE TITLE
Failed to install flatpak theme

### ISSUE THREAD

**Issue by minhntp** [vinceliuice/WhiteSur-gtk-theme#426]
I have installed all the dependencies required in the result.
But it still cannot install.

![flatpak-theme-error](https://user-images.githubusercontent.com/10728524/133130122-ab999a4a-cd88-4fad-a29f-93d20bdfcf9d.png)

**Comment by vinceliuice** [comment]
Did you installed ostree and appstream-util? `sudo apt install ostree appstream-util`

**Comment by minhntp** [comment]
Yes, I have installed both of them:
ostree is already the newest version (2020.3-1).
appstream-util is already the newest version (0.7.16-1ubuntu1).

**Comment by vinceliuice** [comment]
I tested on my ubuntu 20.04 but everything works well, did you installed theme before you run `./tweaks.sh -F`

**Comment by minhntp** [comment]
Yes, I have. I have applied the theme to applications and shell.

**Comment by vinceliuice** [comment]
I don't know what's going on, you installed flatpak right？ what commond you run to install this theme？

**Comment by rivanfebrian123** [comment]
@minhntp Could you please add `ld -1d "$theme_path"/* 1>&2 || true` before line 51 in `lib-flatpak.sh` and send us the output?

**Comment by minhntp** [comment]
This is the result: 
![modified-lib-flatpak-sh](https://user-images.githubusercontent.com/10728524/133592798-032009f0-9eeb-49ba-bdcf-52ad64e735f4.png)
![after-modified-lib-flatpak-sh](https://user-images.githubusercontent.com/10728524/133592808-2d017476-517d-470b-9cc4-113e9e36f384.png)

**Comment by rivanfebrian123** [comment]
Oh, sorry, I mean, add `ld -1d "$theme_path"/* 1>&2` (without `|| true`) and then add a newline. So it's gonna be like:
```
ld -1d "$theme_path"/* 1>&2
theme_gtk_version= ...
```

**Comment by minhntp** [comment]
![Screenshot from 2021-09-16 18-24-54](https://user-images.githubusercontent.com/10728524/133603972-629a457b-0af5-4f6c-9cb9-880a97658406.png)
![Screenshot from 2021-09-16 18-25-35](https://user-images.githubusercontent.com/10728524/133603984-05814edd-4a93-41e5-b1cd-49492b5c22fb.png)

**Comment by vinceliuice** [comment]
@minhntp  Isn't it works now? :
https://user-images.githubusercontent.com/10728524/133592808-2d017476-517d-470b-9cc4-113e9e36f384.png

**Comment by minhntp** [comment]
It's in the flatpak app list, but the theme is not applied on flatpak applications.
![Screenshot from 2021-09-16 20-52-54](https://user-images.githubusercontent.com/10728524/133624804-d7cc90de-b6c1-4186-9e7f-38878c7432bc.png)
![Screenshot from 2021-09-16 20-53-45](https://user-images.githubusercontent.com/10728524/133624846-a58a3ea3-5c6f-4350-9584-691226841051.png)

**Comment by minhntp** [comment]
Hey it works on other apps!
Thank you.

![Screenshot from 2021-09-16 20-53-45](https://user-images.githubusercontent.com/10728524/133625646-e40ff2c6-a345-4734-acba-613389c353d9.png)

**Comment by vinceliuice** [comment]
Do not support gtk4 apps, only gtk3 app can use the flatpak version

**Comment by minhntp** [comment]
Got it. Thanks!

**Comment by OlaoluwaM** [comment]
Same issue, but since it's not supported will look for a workarounf
